### ISSUE TITLE
How to stop submitted RODPS query command

### ISSUE THREAD

**Issue by danzhuibing** [yaoweidong/aliyun-odps-r-plugin#1]
I use RODPS in RStudio Server. When I submit a RODPS SQL query command, the whole console will be blocked until the query result returns. You know, **it is really common that ODPS resource becomes limited since so many users are in the same pool**. In this case, it will take a long time until we get the result from a submitted RODPS SQL query command. So I just wonder **if there is a way to kill the submitted RODPS SQL query command instead of waiting for it**. It seems that the red stop button in the console is out of work.

**Comment by lyman** [comment]
Good point. I'm afraid there's no convenient way to stop running SQL in RODPS now.

However, this could be done elsewhere.

Prepare a [ODPS Console](http://repo.aliyun.com/download/odpscmd/0.23.1/odpscmd_public.zip) and type `show p` to list all today's instances, and type `kill instance_id` to stop a specific SQL instance.

BTW, it would be nice to file issue at http://github.com/aliyun/aliyun-odps-r-plugin, which is official.

**Comment by cobbxia** [comment]
需要定制Rstudio方式解决
